libs/hdf-converters/test/mappers/forward/snyk_mapper.spec.ts

```diff
@@ -25,16 +33,23 @@ describe('snyk_mapper', () => {
       )
     );
   });
-  it('Successfully converts Snyk cli targeted at a remote/online repository data', () => {
-    const mapper = new SnykMapper(
-      JSON.parse(
-        fs.readFileSync(
-          'sample_jsons/snyk_mapper/sample_input_report/nodejs-goof-remote.json',
-          {encoding: 'utf-8'}
-        )
+  it('Successfully converts Snyk cli targeted at a remote/online repository data', async () => {
```

Comment:
Can we make this a mock API? See `heimdall2/test/support/server`

Reply:
the two tests currently in there are both for the file based inputs (the results are different if the cli was pointed at a local copy vs a remote).  i can add a test for the api mapper using a mock